language-server/src/server.js

```diff
@@ -31,10 +31,26 @@ import { addWorkspaceFolders, workspaceSchemas, removeWorkspaceFolders, watchWor
 import { getSemanticTokens } from "./semantic-tokens.js";
 import { JsonSchemaDocument } from "./json-schema-document.js";
 import * as Instance from "./json-instance.js";
+import picomatch from "picomatch";
 
 
 setShouldValidateSchema(false);
 
+let schemaFilePatterns = ["**/*.schema.json", "**/schema.json"];
```

Comment:
Is it possible to fold this into configuration? I think we can put this into `globalSettings`. Then we should be getting the value only from configuration.